x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_summary_view.tsx

```diff
@@ -7,171 +7,335 @@
+}) => (
+  <EuiFlexGroup direction="row" gutterSize="none" alignItems="center">
+    <EuiFlexItem>
+      <ThreatSummaryPanelTitle>{title}</ThreatSummaryPanelTitle>
+    </EuiFlexItem>
+    <EuiFlexItem grow={false}>
+      <EuiToolTip title={title} content={toolTipContent}>
+        <EuiIcon type="iInCircle" size="m" />
+      </EuiToolTip>
+    </EuiFlexItem>
+  </EuiFlexGroup>
+);
+
+const ThreatSummaryEnrichmentData: React.FC<{
+  browserFields: BrowserFields;
+  data: TimelineEventsDetailsItem[];
+  enrichments: CtiEnrichment[];
+  timelineId: string;
+  eventId: string;
+  isDraggable?: boolean;
+}> = ({ browserFields, data, enrichments, timelineId, eventId, isDraggable }) => {
+  const parsedEnrichments = enrichments.map((enrichment, index) => {
+    const { field, type, provider, value } = getEnrichmentIdentifiers(enrichment);
     const eventData = data.find((item) => item.field === field);
     const category = eventData?.category ?? '';
     const browserField = get([category, 'fields', field ?? ''], browserFields);
 
-    const fieldsData = {
-      field,
+    const fieldsData: FieldsData = {
+      field: field ?? '',
       format: browserField?.format ?? '',
       type: browserField?.type ?? '',
-      isObjectArray: eventData?.isObjectArray,
+      isObjectArray: eventData?.isObjectArray ?? false,
     };
 
     return {
-      title: {
-        title: field,
-        type,
-      },
-      description: {
-        eventId,
-        index,
-        provider,
-        timelineId,
-        value,
-        data: fieldsData,
-        browserField,
-      },
+      fieldsData,
+      type,
+      provider,
+      index,
+      field,
+      browserField,
+      value,
     };
   });
+
+  const [investigation, indicator] = partition(parsedEnrichments, ({ type }) =>
+    isInvestigationTimeEnrichment(type)
+  );
+
+  return (
+    <>
+      {indicator.length > 0 && (
+        <>
+          <EuiPanel hasBorder paddingSize="s" grow={false}>
+            <ThreatSummaryPanelHeader
+              title={i18n.INDICATOR_ENRICHMENT_TITLE}
+              toolTipContent={i18n.INDICATOR_TOOLTIP_CONTENT}
+            />
+
+            {indicator.map(({ fieldsData, index, field, provider, browserField, value }) => (
+              <EnrichedDataRow
+                key={field}
+                field={field}
+                value={
+                  <EnrichmentDescription
+                    eventId={eventId}
+                    index={index}
+                    provider={provider}
+                    timelineId={timelineId}
+                    value={value}
+                    data={fieldsData}
+                    browserField={browserField}
+                    isDraggable={isDraggable}
+                  />
+                }
+              />
+            ))}
+          </EuiPanel>
+        </>
+      )}
+
+      {indicator.length > 0 && investigation.length > 0 && <EuiSpacer size="m" />}
+
+      {investigation.length > 0 && (
```

Comment:
`indicator` and `investigation` pieces seem to be rendered almost identically. We could consider adding a tiny ThreatEnrichments component that would take in the `title`, `tooltipContent`, and `items` as props so that we can keep this area more DRY

Reply:
i like this idea